Gemfile.lock

```diff
@@ -2,6 +2,7 @@ PATH
   remote: .
   specs:
     killbill-cybersource (4.0.11)
+      accent_to_ascii (~> 0.1.1)
```

Comment:
why not use the existing gem? there are still a few lines of code if we unpack them from the gem.